tests/test-pool-search.rb

```diff
@@ -165,7 +165,46 @@
   #
   # test that a date range includes an expected item
   #
-  #
+  it "#{url} should return one or more items" do
+    post search_endpoint, { :query => "author:{jefferson}", :pagination => { :start => 0, :rows => 1 }  }
```

Comment:
If we're using a stock query across these tests, we should make it a variable that's reused across tests.  I'd prefer something guaranteed to return results (and one that probably returns different results as time goes on) but as long as we parameterize it, we can easily address that concern later.